Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM python:3.12.3-slim
+LABEL maintainer="94nj111@gmail.com"
+
+ENV PYTHONDONTWRITEBYTECODE=True
+ENV PYTHONUNBUFFERED=True
+
+WORKDIR /app
+
+ARG UID=10001
+RUN adduser \
+    --disabled-password \
+    --gecos "" \
+    --home "/app" \
+    --shell "/sbin/nologin" \
+    --no-create-home \
+    --uid "${UID}" \
+    appuser
```

Comment:
Consider creating the home directory for `appuser` by removing the `--no-create-home` option. This ensures that the user has a valid home directory, which might be necessary for some applications.